libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -31,27 +31,10 @@ const
   GossipSubCodec* = "/meshsub/1.1.0"
   GossipSubCodec_10* = "/meshsub/1.0.0"
 
-# overlay parameters
-const
```

Comment:
I think we can keep the consts as they are in the default spec and set them as defaults, overridable in `init()`?

Reply:
True